xds/internal/xdsclient/lds_test.go

```diff
@@ -1222,6 +1226,7 @@ func (s) TestUnmarshalListener_ServerSide(t *testing.T) {
 																IdentityCertName:     "identityCertName",
 																RequireClientCert:    true,
 															},
+															HTTPFilters: []HTTPFilter{},
```

Comment:
I thought this isn't necessary? There should be an option to tell `cmp.Diff` that `nil` is the same as an empty slice.

Reply:
And add a test with actually http filters.

Reply:
How can I test http filters if the ServerInterceptor isn't defined yet? There's also no concrete HTTP Filters, like fault injection on client side.